docs/content/installation/using-aws-marketplace-image.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Using the AWS Marketplace Ingress Controller Image
+description: 
+weight: 2300
+doctypes: [""]
+toc: true
+---
+
+This document will walk you through the steps needed to use the NGINX Ingress Controller through the AWS Marketplace. There are additional steps that must be followed in order for the AWS Marketplace NGINX Ingress Controller to work properly.
+
+> **IMPORTANT**: This document uses EKS version 1.19. EKS versions < 1.19 require additional security settings within the NGINX Pod to work properly with marketplace images. 
+> This document discusses using eksctl to perform necessary steps to enable the Kubernetes cluster access to deploy NGINX Ingress Controller from the Marketplace. Please make sure you are running a newer version of eksctl and AWS cli.
+
+> **NOTE**: NGINX Ingress controller from the Marketplace does NOT work in AWS Region US-West-1.
+
+## Instructions
+Instructions for using AWS Marketplace:
+
+1. Ensure you have a working AWS EKS cluster. If you do not have a EKS cluster, you can create one using either the AWS console, or using the AWS tool eksctl. See [this guide](https://docs.aws.amazon.com/EKS/latest/userguide/getting-started-eksctl.html) for details on getting started with EKS using eksctl.
```

Comment:
The link doesn't work. Perhaps this is the intended one? (case sensitive it seems)
https://docs.aws.amazon.com/eks/latest/userguide/getting-started-eksctl.html